commands/bootstrap/resources/topology/topology_infrastructure_openstack.tmpl

```diff
@@ -1,6 +1,7 @@
 {{ define "Infrastructure" }}
-      type: org.ystia.yorc.infrastructure.OpenStackConfig
-      properties: {{formatAsYAML .Infrastructures.openstack 8}}
+      type: org.ystia.yorc.location.OpenStackConfig
+      properties: {{formatAsYAML .Location.Properties 8}}
+        location_name: "{{.Location.Name}}"
```

Comment:
Why `location_name` appear for openstack and not for others?

Reply:
yes it needs to appear on all